packages/synapse-interface/pages/state-managed-bridge/index.tsx

```diff
@@ -174,6 +166,20 @@ const StateManagedBridge = () => {
         (quote) => quote.bridgeModuleName === 'SynapseRFQ'
       )
 
+      let quote
+
+      if (rfqQuote) {
+        quote = rfqQuote
+      } else {
+        quote = await synapseSDK.bridgeQuote(
```

Comment:
You can go even further and strip this call altogether.

allBridgeQuotes returns the full list of quotes sorted by destination amount out, so picking the first entry from the list should effectively do the same thing as issuing another bridgeQuote

Reply:
This reminds me that SDK documentation hasn't been updated for a loooong time - aiming to do this next week

Reply:
Updated [here](https://github.com/synapsecns/sanguine/pull/1861/commits/0adc35cb771bd4d59ed61e0442b3e883503c09a6)